modules/ossm-cr-tracing-jaeger.adoc

```diff
@@ -31,7 +30,7 @@ This example illustrates tracing parameters for the {ProductName} custom resourc
 |`hub`
 | The hub that the Operator uses to pull Jaeger images
 | A valid image repo
-| `jaegertracing/` or `registry.redhat.io/openshift-istio-tech-preview/`
+| `jaegertracing/` or `registry.redhat.io/openshift-service-mesh/`
```

Comment:
its not `registry.redhat.io/openshift-service-mesh` but `registry.redhat.io/distributed-tracing`.

Don't think the values should have the `/` at the end, as the template includes that.

Reply:
This entire file was rewritten and is now in the ossm-configuring-jaeger.adoc file.